video_xblock/static/js/studio-view/studio-edit-transcripts-autoload.js

```diff
@@ -78,31 +62,29 @@ function createAvailableTranscriptBlock(defaultTranscript, initialDefaultTranscr
     var langLabel = defaultTranscript.label;
     var initialDefaultTranscripts = initialDefaultTranscriptsData[0];
     var initialDefaultTranscriptsLangCodes = initialDefaultTranscriptsData[1];
-    // Get all the currently available transcripts
-    var allAvailableTranscripts = getDefaultTranscriptsArray('available');
     // Create a new available transcript if stored on a platform and doesn't already exist on video xblock
-    var isNotDisplayedAvailableTranscript = $.inArray(langCode, allAvailableTranscripts) === -1;
+    var isNotDisplayedAvailableTranscript = $.inArray(langCode, getDefaultTranscriptsArray('available')) === -1;
     var isStoredVideoPlatform = $.inArray(langCode, initialDefaultTranscriptsLangCodes) !== -1;
-    var $availableLabel;
-    var isHiddenAvailableLabel;
-    var $newAvailableTranscriptBlock;
-    var downloadUrlApi;
     if (isNotDisplayedAvailableTranscript && isStoredVideoPlatform) {
         // Show label of available transcripts if no such label is displayed
-        $availableLabel = $('div.custom-field-section-label.available-transcripts');
-        isHiddenAvailableLabel = !$('div.custom-field-section-label.available-transcripts:visible').length;
-        if (isHiddenAvailableLabel) { $availableLabel.removeClass('is-hidden'); }
+        if (!$('div.custom-field-section-label.available-transcripts:visible').length) {
+            $('div.custom-field-section-label.available-transcripts').removeClass('is-hidden');
+        }
         // Create a default (available) transcript block
-        $newAvailableTranscriptBlock = $('.available-default-transcripts-section:hidden').clone();
-        $newAvailableTranscriptBlock.removeClass('is-hidden').appendTo($('.default-transcripts-wrapper'));
+        $('.available-default-transcripts-section:hidden')
+            .clone()
```

Comment:
> why do we clone it?

We can have several elements like this. So @OPersian created one which is used as template and add copied it to each new transcript if needed. This kind realisation was taken from base video xmodule.